resources/lang/pl/forum.php

```diff
@@ -24,95 +24,234 @@
         'create' => [
             '_' => 'Ustaw tło nagłówka',
             'button' => 'Dodaj tło',
-            'info' => 'Nagłówek powinien być w rozdzielczości :dimensions. Możesz także upuścić swoje tło tutaj, aby je dodać.',
+            'info' => 'Nagłówek powinien mieć rozdzielczość :dimensions. Możesz upuścić tutaj swoje tło, aby je dodać.',
         ],
 
         'destroy' => [
             '_' => 'Usuń tło nagłówka',
-            'confirm' => 'Czy jesteś pewien, że chcesz usunąć tło nagłówka?',
+            'confirm' => 'Na pewno chcesz usunąć tło nagłówka?',
         ],
     ],
-    'pinned_topics' => 'Przypięte tematy',
+
+    'email' => [
+        'new_reply' => '[osu!] Nowa odpowiedź dla wątku ":title"',
+    ],
+
+    'forums' => [
+        'topics' => [
+            'empty' => 'Brak wątków!',
+        ],
+    ],
+
+    'pinned_topics' => 'Przypięte wątki',
     'post' => [
-        'confirm_delete' => 'Na pewno usunąć post?',
-        'edited' => 'Ostatnio edytowane przez :user o :when, edytowane :count razy łącznie.',
-        'posted_at' => 'dodane :when',
+        'confirm_destroy' => 'Na pewno usunąć wątek?',
+        'confirm_restore' => 'Na pewno przewrócić wątek?',
+        'edited' => 'Ostatnio edytowe przez :user :when, łącznie edytowane :count razy.',
+        'posted_at' => 'opublikowane :when',
         'actions' => [
-            'delete' => 'Usuń post',
-            'edit' => 'Edytuj post',
+            'destroy' => 'Usuń wątek',
+            'restore' => 'Przywróć wątek',
+            'edit' => 'Edytuj wątek',
         ],
     ],
     'search' => [
-        'go_to_post' => 'Przejdź do posta',
-        'post_number_input' => 'wpisz numer posta',
-        'total_posts' => 'Łączna ilość postów: :posts_count',
+        'go_to_post' => 'Przejdź do wątku',
+        'post_number_input' => 'wprowadź numer wątku',
```

Comment:
wątek = thread to znaczy, że wątek to pierwszy post i wszyskie odpowiedzi, więc raczej powinieneś w tym przypadku zamienić wątku na posta